### PR TITLE
35 comparison feature in business applications and component list redirects to 500 error page

### PR DESCRIPTION
**Pull request by expeditionengineer**

### CODE REVIEW

**Comment by expeditionengineer** [comment]
- solved error in comparison page of component-list
- included TestComponentList into CI test execution

File: webcentral/src/webcentral_app/static/js/toolComparison-feature.js
Comment (c0nb4) [inline webcentral/src/webcentral_app/static/js/toolComparison-feature.js:1]: Can I delete this (the commented code)?

Comment (expeditionengineer) [reply webcentral/src/webcentral_app/static/js/toolComparison-feature.js:1]: i removed them

**Review by c0nb4 — APPROVED** [review]

**Comment by c0nb4** [comment]
Selenium tests are failing: 

```
======================================================================
ERROR: testLinksFromOverviewPage (test.scripts.test_component_list.TestComponentList)
Test if the links from the negative environmental impacts page leads to a working page.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/wissen-digital-ewb/wissen-digital-ewb/webcentral/test/06_system_test/test/scripts/test_component_list.py", line 139, in testLinksFromOverviewPage
    self.waitUntilConditionIsMet(
  File "/home/runner/work/wissen-digital-ewb/wissen-digital-ewb/webcentral/test/06_system_test/src/test_base/webdriver_setup.py", line 273, in waitUntilConditionIsMet
    wait.until(functionHandler)
  File "/home/runner/work/wissen-digital-ewb/wissen-digital-ewb/webcentral/test/testing_venv/lib/python3.10/site-packages/selenium/webdriver/support/wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
```

**Pull request merged**